File: src/main/java/no/nav/veilarboppfolging/domain/AvslutningStatusData.java
package no.nav.veilarboppfolging.domain;

import lombok.Builder;
import lombok.Data;

import java.time.LocalDate;

@Data
@Builder
public class AvslutningStatusData {
    public final boolean kanAvslutte;
    public final boolean underOppfolging;
    public final boolean harYtelser;
    public final boolean underKvp;
    public final LocalDate inaktiveringsDato;
    public final boolean erIserv;
    public final boolean harAktiveTiltaksdeltakelser;
}

File: src/main/kotlin/no/nav/veilarboppfolging/domain/AvslutningStatusData.kt
package no.nav.veilarboppfolging.domain

import java.time.LocalDate

data class AvslutningStatusData(
    val kanAvslutte: Boolean,
    val underOppfolging: Boolean,
    val harYtelser: Boolean,
    val underKvp: Boolean,
    val inaktiveringsDato: LocalDate?,
    val erIserv: Boolean,
    val harAktiveTiltaksdeltakelser: Boolean,
    val erDeltakerIUngdomsprogrammet: Boolean,
    val erArbeidssoeker: Boolean,
    val harAap: Boolean,
)